src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    border-radius: 0.5px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
not fixed comment about correct work for all modifiers
it means to add selectors for all cases of amount active stars
![Screenshot from 2023-06-05 13-08-01](https://github.com/mate-academy/layout_product-cards/assets/33029864/d3c22dfa-ffa1-4c2c-9f9b-187d49fa6aff)